CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Incorrect modifiers at startup on X11
 - `Add` and `Subtract` keys are now named `NumpadAdd` and `NumpadSubtract` respectively
 - Feature checking when cross compiling between different operating systems
+- Crash when write to clipboard fails on Wayland
```

Comment:
```suggestion
- Crash when writing to the clipboard fails on Wayland
```

I don't think the noun write is wrong here, but I think using the verb reads a little nicer.